client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	'Business services': i18n.translate( 'Business services' ),
+	CBD: i18n.translate( 'CBD' ),
+	CCTV: i18n.translate( 'CCTV' ),
+	Cafe: i18n.translate( 'Cafe' ),
+	Cake: i18n.translate( 'Cake' ),
+	Cakes: i18n.translate( 'Cakes' ),
+	Cancer: i18n.translate( 'Cancer' ),
+	Cannabis: i18n.translate( 'Cannabis' ),
+	'Car audio': i18n.translate( 'Car audio' ),
+	'Car repairs': i18n.translate( 'Car repairs' ),
+	Cars: i18n.translate( 'Cars' ),
+	Cats: i18n.translate( 'Cats' ),
+	Celebrities: i18n.translate( 'Celebrities' ),
+	Ceramics: i18n.translate( 'Ceramics' ),
+	Charity: i18n.translate( 'Charity' ),
+	'Child abuse': i18n.translate( 'Child abuse' ),
+	Childbirth: i18n.translate( 'Childbirth' ),
+	'Chinese restaurant': i18n.translate( 'Chinese restaurant' ),
+	Choir: i18n.translate( 'Choir' ),
+	Christ: i18n.translate( 'Christ' ),
+	Christianity: i18n.translate( 'Christianity' ),
+	Church: i18n.translate( 'Church' ),
+	'Classified ads': i18n.translate( 'Classified ads' ),
+	'Cleaning services': i18n.translate( 'Cleaning services' ),
+	'Climate change': i18n.translate( 'Climate change' ),
+	Clinic: i18n.translate( 'Clinic' ),
+	Clothes: i18n.translate( 'Clothes' ),
+	Clothing: i18n.translate( 'Clothing' ),
+	Coach: i18n.translate( 'Coach' ),
+	Coaching: i18n.translate( 'Coaching' ),
+	Coding: i18n.translate( 'Coding' ),
+	Coffee: i18n.translate( 'Coffee' ),
+	'Coffee shop': i18n.translate( 'Coffee shop' ),
+	College: i18n.translate( 'College' ),
+	Comedy: i18n.translate( 'Comedy' ),
+	Commercial: i18n.translate( 'Commercial' ),
+	Communication: i18n.translate( 'Communication' ),
+	Community: i18n.translate( 'Community' ),
+	'Community association': i18n.translate( 'Community association' ),
+	'Community services': i18n.translate( 'Community services' ),
+	Computer: i18n.translate( 'Computer' ),
+	'Computer repairs': i18n.translate( 'Computer repairs' ),
+	Computing: i18n.translate( 'Computing' ),
+	'Concrete restorations': i18n.translate( 'Concrete restorations' ),
+	Conduits: i18n.translate( 'Conduits' ),
+	Conference: i18n.translate( 'Conference' ),
+	Construction: i18n.translate( 'Construction' ),
+	'Construction engineer': i18n.translate( 'Construction engineer' ),
+	Consultant: i18n.translate( 'Consultant' ),
+	Consulting: i18n.translate( 'Consulting' ),
+	'Consulting agency': i18n.translate( 'Consulting agency' ),
+	'Consulting business': i18n.translate( 'Consulting business' ),
+	'Consulting engineering': i18n.translate( 'Consulting engineering' ),
+	Consumables: i18n.translate( 'Consumables' ),
+	'Content creation': i18n.translate( 'Content creation' ),
+	Cooking: i18n.translate( 'Cooking' ),
+	Copywriting: i18n.translate( 'Copywriting' ),
+	'Corporate services': i18n.translate( 'Corporate services' ),
+	'Corrosion engineering': i18n.translate( 'Corrosion engineering' ),
+	Cosmetics: i18n.translate( 'Cosmetics' ),
+	Cosplay: i18n.translate( 'Cosplay' ),
+	Councelling: i18n.translate( 'Councelling' ),
```

Comment:
Also not sure about this one. Supposed to be the British spelling or simply misspelled "Counseling"?

Reply:
Yeah, I was using British spelling. Let's change it to `Counselling`.